docs/guides/high-availability/examples/generated/keycloak-ispn.yaml

```diff
@@ -449,12 +451,15 @@ spec:
   startOptimized: false # <2>
   features:
     enabled:
-      - multi-site # <3>
```

Comment:
Don't we need this?

Reply:
Not in this document but yes we need to add it to keycloak.yaml. I will address this